e2e/src/usecases/Assets.js

```diff
@@ -99,75 +108,79 @@ export default Assets = () => {
-          await element(by.id('TokenDetailsMoreActions')).swipe('down')
-          await waitForElementId('TokenDetails/AssetValue')
-        })
-      }
+        if (moreActions.includes('Send')) {
+          it('send action under more actions navigates to send flow', async () => {
+            await element(by.id('TokenDetails/Action/More')).tap()
+            await waitForElementByIdAndTap('TokenDetailsMoreActions/Send')
+            await validateSendFlow(symbol)
+            await element(by.id('TokenDetailsMoreActions')).swipe('down')
+            await waitForElementId('TokenDetails/AssetValue')
+          })
+        }
 
-      if (moreActions.includes('Withdraw')) {
-        it('withdraw action under more actions navigates to withdraw spend screen', async () => {
-          await element(by.id('TokenDetails/Action/More')).tap()
-          await waitForElementByIdAndTap('TokenDetailsMoreActions/Withdraw')
-          await waitForElementId('FiatExchangeTokenBalance')
-          await element(by.id('BackChevron')).tap()
-          await element(by.id('TokenDetailsMoreActions')).swipe('down')
-          await waitForElementId('TokenDetails/AssetValue')
-        })
-      }
+        if (moreActions.includes('Add')) {
+          it('add action under more actions navigates to add cico flow', async () => {
+            await element(by.id('TokenDetails/Action/More')).tap()
+            await waitForElementByIdAndTap('TokenDetailsMoreActions/Add')
+            await validateAddFlow(symbol)
+            await element(by.id('TokenDetailsMoreActions')).swipe('down')
+            await waitForElementId('TokenDetails/AssetValue')
+          })
+        }
 
-      if (learnMore) {
-        it('learn more navigates to coingecko page', async () => {
-          await waitForElementByIdAndTap('TokenDetails/LearnMore')
-          await waitForElementId('RNWebView')
-          await waitFor(element(by.text('www.coingecko.com')))
-            .toBeVisible()
-            .withTimeout(10 * 1000)
-          await element(by.id('WebViewScreen/CloseButton')).tap()
-          await waitForElementId('TokenDetails/AssetValue')
+        if (moreActions.includes('Withdraw')) {
+          it('withdraw action under more actions navigates to withdraw spend screen', async () => {
+            await element(by.id('TokenDetails/Action/More')).tap()
+            await waitForElementByIdAndTap('TokenDetailsMoreActions/Withdraw')
+            await waitForElementId('FiatExchangeTokenBalance')
+            await element(by.id('BackChevron')).tap()
+            await element(by.id('TokenDetailsMoreActions')).swipe('down')
+            await waitForElementId('TokenDetails/AssetValue')
+          })
+        }
+
+        if (learnMore) {
+          it('learn more navigates to coingecko page', async () => {
+            await scrollIntoView(`Learn more about ${name}`, 'TokenDetailsScrollView')
+            await waitForElementByIdAndTap('TokenDetails/LearnMore')
```

Comment:
optional nit: seems odd to use text for scrolling and id for tapping. would be nice if this was consistent